### ISSUE TITLE
Optimize tacking option

### ISSUE THREAD

**Issue by theeko74** [seandepagnier/weather_routing_pi#186]
@seandepagnier 
Can you please explain what's the option "Optimize tacking" is doing?
In this example, you can see that when the option is activated, my boat is sailing straight into the wind, which is not possible unless you use the engine...

<img width="1169" alt="capture d ecran 2018-04-02 a 23 36 15" src="https://user-images.githubusercontent.com/22415254/38218908-a39078c8-36d3-11e8-91ec-5626d8f67c04.png">

When I disable this option, it comes back to normal routing. We can clearly see that sailing straight into the wind is not possible.

<img width="772" alt="capture d ecran 2018-04-02 a 23 36 25" src="https://user-images.githubusercontent.com/22415254/38218926-c0c25326-36d3-11e8-9e09-da69a8f27b0b.png">

I'm asking that because I am thinking about implementing an "engin option" that could be use bellow a define speed and/or to navigate upwind when appropriate.

**Comment by seandepagnier** [comment]
I think this is explained in the wiki and documentation is it not?

optimize tacking assumes you tack periodically to maintain the course
making tacks more often than the isochron.


Please do not implement an engine option.  Just make an engine polar
and use it along with the sailing polars.

**Comment by theeko74** [comment]
Yes, shorter than the isochrons, but how can I sail with 0° from apparent wind?! Does it mean than during isochron 0 and 1, I will do several tackings (which in fact will take much road and much more time than the calculated 1h).

You're right implementing a polar with engine will solve my issue, thanks.

By the way, I've looked at lib for nice graph but I did not find something yet. If you see something, I am still interested to help on plot improvement.
